### DISCUSSION TITLE
百度與小馬智行取得中國第一個自駕計程車許可，將在北京上路

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#581] [General]
**北京市政府開出了中國第一個自駕車試點許可，兩家獲准上路的公司分別是網路巨頭百度，以及自動駕駛新創 Pony.ai，雙方都將在本週讓自駕計程車上路載客。**

太牛啦，但是這樣計程車司機會不會被壓縮到，如果夠安全的話，就要比拚是原本的人力比較便宜或是自駕車比較便宜，現在價格還是主要的選擇標準。

全文 : https://technews.tw/2022/04/28/baidu-ponyai-robotaxi-china/

**Comment by PoooYen** [comment]
中國的科技普及和進步已經海放台灣了.......

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，我們也要好好加油。

**Comment by cordydeng** [comment]
文中提到仍然會有駕駛員在，可見自駕車技術還沒到非常完善，不過就算未來技術成熟了，我應該也不敢搭自駕車啦

**Comment by JSHT** [comment]
實際上路，突發狀況的情形太多了，想達成真自駕還有好長一段路走。
目前除特斯拉之外，絕大多數自駕車的開發都是採用 [Nvidia 開發的解決方案](https://www.nvidia.com/zh-tw/self-driving-cars/)。希望在競爭下能夠有更快速的進展

**Comment by HsuChen88** [comment]
如果所有的車都是電腦自架的話路上的突發事件或許就能減少了
但也可能像電影裡一樣
所有車輛都被一人控制 造成混亂

**Comment by handsomeniko** [comment]
自駕車真的指日可待

**Comment by ThomasMeng0515** [comment]
之前在美國其實很早就有計程車自駕車在試驗了，但那時候也需要工程師在車上監督，若有什麼即時狀況便可以馬上處理，希望以後可以完全作到無人監督的自駕計程車，這樣會方便許多

**Comment by yenwei12138** [comment]
文中說到，為了彰顯是自駕車，安全人員要坐在副駕而不是駕駛座，這感覺很沒必要又壓縮了載客空間。

**Comment by Ben6uo** [comment]
謝謝分享，但是距離完全的靠AI來自動操作的日子還有一段時間

**Comment by henrywen547** [comment]
這樣的技術還沒有很成熟，確實會造成不少的疑慮，搭計程車卻不能隨心所欲選擇目的地且須有安全人員
與其說開始上路，到不如說是拿一般人來做自駕車的實驗對象

**Comment by Sazmafuji** [comment]
願，不會發生太大事故

**Comment by Lai27330** [comment]
好有趣的功能，但感覺如果還是要有安全人員在的話，那跟輔助駕駛和司機的功能感覺相差不大。

**Comment by HaoFish123** [comment]
感謝分享，希望自駕車能盡快實現。

**Comment by treeann** [comment]
期待看到實體

**Comment by yucheng1028** [comment]
中國開放技術的速度真的比台灣快太多了

**Comment by alexthebigpp** [comment]
這樣中國失業人口又要更多了
